Dockerfile

```diff
@@ -50,14 +50,14 @@ ENV VERSION_LAST_TAG=$VERSION_LAST_TAG
 # - https://packages.debian.org/bookworm/openjdk-17-jdk
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-    docker.io=20.10.24+dfsg1-1+b3 \
-    libssl-dev=3.0.11-1~deb12u2 \
-    pkg-config=1.8.1-1 \
-    unzip=6.0-28 \
-    wget=${WGET_VERSION} \
-    software-properties-common=0.99.30-4 \
+    docker.io \
+    libssl-dev \
+    pkg-config \
+    unzip \
+    wget \
+    software-properties-common \
   && apt-get install -y --no-install-recommends \
-    openjdk-17-jdk=${OPENJDK_17_VERSION} \
+    openjdk-17-jdk \
```

Comment:
Where is the `${OPENJDK_17_VERSION}` format specified? and the `${WGET_VERSION}`? We should probably remove those too?

In fact, I'd like to look into the reason these were parametrized? (maybe a git blame or a slack search) to check we're not creating problems here?